solr/solr-ref-guide/modules/indexing-guide/pages/external-files-processes.adoc

```diff
@@ -21,6 +21,12 @@ It can also consume a stream of tokens that have already undergone analysis with
 
 == The ExternalFileField Type
 
+.ExternalFileField has been Deprecated
+[WARNING]
+====
+ExternalFileField has been deprecated and will be removed in a future release.
```

Comment:
[0] Might be worth pointing to the alternative here, for folks that don't find the javadocs link?

Reply:
A comment referring to in-place updates of numeric fields